tasks/install-from-source.yml

```diff
@@ -1,27 +1,39 @@
 ---
+
+- name: Include OS-specific variables (RedHat).
+  include_vars: "{{ ansible_os_family }}.yml"
+  when:
+  - ansible_os_family == "RedHat"
+  - ansible_distribution != "Fedora"
+  - git_install_from_source_dependencies is not defined
```

Comment:
D'oh, one other thing I just noticed—you can go ahead and remove the `git_install_from_source_dependencies is not defined` condition from each of these—no harm in having the `__` versions of the variables defined (IMO), and it removes 3 lines from the YAML file, which means easier to scan through an already kinda-long task file.